### PR TITLE
Support toFormat('jpg')

### PR DESCRIPTION
**Pull request by jingsam**

### CODE REVIEW

**Comment by lovell** [comment]
Hello, thanks for this, please can you fix the linting errors.

**Comment by lovell** [comment]
Please can you add a unit test for this.

**Comment by jingsam** [comment]
It seems there is no independent unit test for `output.js`. Where should I start writing a test for this?

**Comment by lovell** [comment]
Thanks for the update, please add a test to [test/unit/io.js](https://github.com/lovell/sharp/blob/master/test/unit/io.js) for now (sorry that file is so big, it needs breaking up).

File: test/unit/io.js [hunk 1]
```diff
@@ -253,6 +253,21 @@ describe('Input/output', function () {
       });
   });
 
+  it('Support output to jpg format', function (done) {
+    sharp(fixtures.inputJpg)
```
Comment (lovell) [inline test/unit/io.js]: Is it worth selecting an input image in a format other than JPEG?

**Comment by lovell** [comment]
Thank you!

**Pull request merged**